src/ocamlorg_frontend/pages/exercises.eml

```diff
@@ -1,23 +1,59 @@
+let exercises_sidebar_link
+~title
+~href
+~current =
+  match current with
+  | true ->
+    <a class="block text-primary-600 bg-primary-100 border-l-2 border-primary-600 py-2 px-3 font-semibold text-sm rounded-sm" href="<%s href %>">
+      <%s title %>
+    </a>
+  | false ->
+    <a class="block hover:text-primary-600 hover:bg-primary-100 hover:border-l-2 hover:border-primary-600 py-2 px-3 font-medium text-sm rounded-sm" href="<%s href %>">
+      <%s title %>
+    </a>
+
+
+let exercises_sidebar_link_group
+?(extra_html="")
```

Comment:
```suggestion
```

parameter appears useless since the function is never called with this parameter